NHLBI_BioData_Catalyst/python/3_PheWAS.ipynb

```diff
@@ -55,7 +67,7 @@
    },
    "outputs": [],
    "source": [
-    "import sys\n",
+    "### Uncomment this code if you are not using the PIC-SURE environment in Seven Bridges, or if you do not have all the necessary dependencies installed.\n",
     "!{sys.executable} -m pip install -r requirements.txt"
```

Comment:
Should comment this line out to match the other notebooks.

Reply:
Resolved in [1765bf34ae1e172d804a6f6293f3c1d40503600f](https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/commit/1765bf34ae1e172d804a6f6293f3c1d40503600f)